src/main.ts

```diff
@@ -21,10 +21,11 @@ export const main = async () => {
       const date = cursor.format("YYYY-MM-DD");
       console.log(`i: ${i}; date: ${date}`);
       const zuoraDataFiles = await cycleDataFilesFromZuora(Stage, zuoraBearerToken, date);
-      const subscriptions1 = subscriptionsDataFileToSubscriptions(zuoraDataFiles.subscriptionsFile);
+      const subscriptions1a = subscriptionsDataFileToSubscriptions(zuoraDataFiles.subscriptionsFile);
+      const subscriptions1b = retainCorrectSubscriptions(subscriptions1a)
       const holidaySubscriptionNames = holidayNamesDataFileToNames(zuoraDataFiles.holidayNamesFile);
       console.log(holidaySubscriptionNames);
-      const subscriptions2 = excludeHolidaySubscriptions(subscriptions1, holidaySubscriptionNames);
+      const subscriptions2 = excludeHolidaySubscriptions(subscriptions1b, holidaySubscriptionNames);
```

Comment:
what does 1a, 1b and 2 mean?

Reply:
https://github.com/guardian/national-delivery-fulfilment/pull/8